conf/logback.xml

```diff
@@ -61,7 +61,7 @@
         <appender-ref ref="STDOUT"/>
     </logger>
 
-    <root level="DEBUG">
+    <root level="ERROR">
```

Comment:
Should this be set to WARN - can only find entry in app-config-staging - was not set in app-config-production